plugins/out_opentelemetry/opentelemetry.h

```diff
@@ -31,7 +31,7 @@
  * log records, and a later batch fails, Fluent Bit will retry ALL the batches,
  * including the ones that succeeded. This is not ideal.
  */
-#define FLB_LOG_RECORD_BATCH_SIZE 1000
+#define DEFAULT_LOG_RECORD_BATCH_SIZE "64"
```

Comment:
isn't "64" too low ?

Reply:
Oh yeah should be fine since it is on the heap now